app/components/common/menu_id/classification/Classification.js

```diff
@@ -1,23 +1,128 @@
-import CLASSIFICATIONS from "../../../lists/classifications";
+import CLASSIFICATION_LABELS from "@/components/enums/classification_labels";
+
+export default function Classification(props) {
+  let percentageAcessibility = props.percentageAcessibility;
+  let percentageDifficulty = props.percentageDifficulty;
+  let percentageEmergency = props.percentageEmergency;
+  let percentageNature = props.percentageNature;
+  let percentageSafety = props.percentageSafety;
+  let percentageShadow = props.percentageShadow;
+  let percentageSupport = props.percentageSupport;
 
-export default function Classification() {
   return (
     <div className="p-8 m-auto">
-      {CLASSIFICATIONS.map((item) => {
-        return (
-          <div key={item.id}>
+      <div>
+        {props.percentageAcessibility ? (
```

Comment:
you should either not create a var called percentageAcessibility or use it in this check

Reply:
what i mean here is

on line 4, you create a var called `acessibility` from `props.percentageAcessibility`

then, on line 15, you use `props.percentageAcessibility`

but then, on line 23, you use `acessibility`

i dont see the need of creating `acessibility`

you should just use always `props.percentageAcessibility`

Reply:
done!